src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+3.4.0
+-----
+
+* Deprecated not using the new `KERNEL_CLASS` server var with `KernelTestCase`.
```

Comment:
If you override `KernelTestCase::createKernel()`, you (probably) don't use `KERNEL_CLASS` but you don't get a deprecation notice, this should be mentioned here as well. As this would make the sentence longer, I think @ro0NL's suggestions makes even more sense.